common/src/main/java/us/ilite/common/config/PracticeBotSystemSettings.java

```diff
@@ -0,0 +1,250 @@
+    public static double llRightBCoeff = -4.53956454545558;
+    public static double llRightCCoeff = -0.0437470770400814;
+    // =============================================================================
+    // Hatch Flower constants
+    // =============================================================================
+    public static double kHatchFlowerSolenoidReleaseTimeSec = 0.250;
+    public static double kHatchFlowerPushDurationSec = 0.250;
+
+    // kHatchFlowerGrabToPushTransitionTimeSec is the time between releasing the 
+    // grab solenoid and engaging the push solenoid.
+    public static double kHatchFlowerGrabToPushTransitionTimeSec = 0.250;
+
+
+    // =============================================================================
+    // 2019 Module Addresses
+    // =============================================================================
+    public static int kCargoSpitLeftSPXAddress = 13;
+    public static int kCargoSpitRightSPXAddress = 14;
+    public static double kCargoSpitSPXCurrentLimit = -1.0;
+
+    public static int kElevatorNEOAddress = -15;
+    // TO-DO: Elevator encoder address?
+    public static int kElevatorNEOEncoderAddress = -1;
+    // public static int kElevatorRedundantEncoderAddress = -1;
+
+    public static int kFourBarNEO1Address = 9;
+    public static int kFourBarNEO2Address = 10;
+    // TO-DO: label solenoid as forward/reverse in spreadsheet
+    public static int kFourBarDoubleSolenoidForwardAddress = 0;
+    public static int kFourBarDoubleSolenoidReverseAddress = 1;
+    public static int kFourBarTBDSensorAddress = -1;
+
+    public static int kHatchFlowerOpenCloseSolenoidAddress = 5;
+    public static int kHatchFlowerExtensionSolenoidAddress = 6;
+
+    public static int kHatchIntakeSPXAddress = 11;
+    public static int kCargoIntakeSPXLowerAddress = 12;
+    // TO-DO DIO spreadsheet empty
+    public static int kIntakeBeamBreakAddress = -1;
+
+    public static int kIntakeWristSRXAddress = 16;
+    // TO-DO Writs encoder addresses?
+    public static int kIntakeWristEncoderA_Address = -1;
+    public static int kIntakeWristEncoderB_Address = -1;
+    public static double kIntakeWristCurrentLimit = -1.0;
+
+    public static int kDriveTrainRightSRX1Address = 2;
+    public static int kDriveTrainRightSPX2Address = 4;
+    public static int kDriveTrainRightSPX3Address = 6;
+    public static int kDriveTrainLeftSRX1Address = 1;
+    public static int kDriveTrainLeftSPX2Address = 3;
+    public static int kDriveTrainLeftSPX3Address = 5;
+
+    /**
+     * This method will take all of the static fields in this class and apply the values to the 
+     * passed in object's static fields that have the exact field name. 
+     * @param destinationObject 
+     *  The object that will have it's static fields updated with the fields in the {@link PracticeBotSystemSettings}
+     */
+    public static void copyOverValues(Object destinationObject) {
```

Comment:
Could this be put into an abstract class rather than directly in a specific class?

Reply:
Yep. I created a new abstract class.